### PR TITLE
[Frontend] Add Leaderboard metric order metadata

### PR DESCRIPTION
**Pull request by Ram81**
### Description

This PR adds feature 2 proposed in #3485 - 

- [x] Show "Higher is better" or "Lower is better" for each leaderboard metric
- [x]  Show metric description in a tooltip when a user hovers on it

Required schema changes are available [here](https://github.com/Cloud-CV/EvalAI-Starters/pull/51)

### CODE REVIEW

**Comment by Ram81** [comment]
Screenshot for reference:

Screen Shot 2021-08-02 at 1 04 41 AM

File: frontend/src/css/modules/leaderboard.scss [hunk 1]
```diff
@@ -67,3 +67,24 @@ span.badge.partial-evaluation {
...
+    font-size: 1.05rem;
+    
+    /* Position the tooltip */
+    position: absolute;
+    z-index: 1;
+}
+
+.leaderboard-label:hover .description {
+    visibility: visible;
+}
```
Comment (RishabhJain2018) [inline frontend/src/css/modules/leaderboard.scss:90]: Please add a new line here.

**Review by RishabhJain2018 — CHANGES_REQUESTED** [review]
@Ram81 What about the models and API changes for this feature?

**Comment by Ram81** [comment]
@RishabhJain2018 we are adding metadata as part of leaderboard schema [here](https://github.com/Cloud-CV/EvalAI-Starters/pull/51) so we won't need any model and API changes. Do we want to add separate field for metadata?

**Comment by RishabhJain2018** [comment]
yeah, that works but I don't see any API changes in the PR to get that value from the config.yaml and then store it?

**Comment by RishabhJain2018** [comment]
Oh, nvm, got it.

File: frontend/src/js/controllers/challengeCtrl.js [hunk 2]
```diff
@@ -876,6 +876,33 @@
...
+                if (metadata[metric] == undefined) {
+                    return false;
+                }
+                return metadata[metric].order_ascending;
+            }
+            return false;
+        };
+
+        vm.getLabelDescription = function(metric) {
+            console.log("");
```
Comment (RishabhJain2018) [inline frontend/src/js/controllers/challengeCtrl.js:893]: Please remove the console from here?

**Review by RishabhJain2018 — CHANGES_REQUESTED** [review]

**Review by RishabhJain2018 — APPROVED** [review]

**Pull request merged**